From 2b0fc239fbe6c5435937c618a282176c6d841e5a Mon Sep 17 00:00:00 2001
From: Vladimir Oltean <olteanv@gmail.com>
Date: Sun, 13 Jan 2019 22:33:40 +0200
Subject: [PATCH 5/5] common: label /sys/devices/virtual/net/* as sysfs_net

* netmgrd, netd, GMS apps try to access nodes here
* Also related:
  https://review.lineageos.org/c/LineageOS/android_device_qcom_sepolicy-legacy/+/230232

Change-Id: I9fba2d45c9bdae7d4b6542934c0790da10ebcd89
Signed-off-by: Vladimir Oltean <olteanv@gmail.com>
---
 common/genfs_contexts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/common/genfs_contexts b/common/genfs_contexts
index cac3200..9858934 100644
--- a/common/genfs_contexts
+++ b/common/genfs_contexts
@@ -18,4 +18,5 @@ genfscon sysfs /devices/soc/soc:qcom,mincpubw/devfreq u:object_r:sysfs_devfreq:s
 genfscon sysfs /kernel/wcd_cpe0 u:object_r:sysfs_audio:s0
 genfscon sysfs /devices/virtual/thermal u:object_r:sysfs_thermal:s0
 genfscon sysfs /devices/virtual/kgsl/kgsl/proc u:object_r:sysfs_kgsl_proc:s0
+genfscon sysfs /devices/virtual/net u:object_r:sysfs_net:s0
 genfscon debugfs /kgsl/proc     u:object_r:kgsl_debugfs:s0
-- 
2.17.1

